### DISCUSSION TITLE
Get total number of excluded lines

### DISCUSSION THREAD

**Discussion by sebastianslutzky** [coverlet-coverage/coverlet#1475] [Ideas]
In .net, when annotating a class or member with ExcludeFromCodeCoverage, there is no way (afaik) to  work out how many lines were ignored. In my organisation, this is a valuable piece of information.
The total lines excludes those excluded, so there is no way to know they exist but were annotated with the attribute.
Is there any way to work out the stat I need from the report?
If not, would you consider such feature (and I can create an Issue for it)

thanks
Sebastian

**Comment by Bertk** [comment]
Please use [ReportGenerator](https://github.com/danielpalme/ReportGenerator) and check whether the created report does not show the missed lines correctly.

<img width="1087" height="413" alt="image" src="https://github.com/user-attachments/assets/c193b70d-2afe-4f72-8da5-6dd2101a6ee6" />

**Comment by sebastianslutzky** [comment]
thanks @Bertk . I've checked this years ago. the line coverage and it wasn't good enough .If i remember wellm it was counting file lines instead of checking coverable code (i.e. blank lines, interfaces, comments, etc)
i would need this line count to be accurate and as early as possible. Coverlet scan sounds like the right place to do so

In my organisation we will probably fork coverlet to add this feature, but it would be nice to include it as an extra column (lines excluded)

**Comment by Bertk** [comment]
coverlet instruments CIL code and empty line will not be handled. Creating a fork is a good choice.

https://dev.to/jarrydpatel/dotnet-basics-what-is-the-common-intermediate-language-586a
